cmd/earthly/secret_cmds.go

```diff
@@ -138,10 +140,17 @@ func (app *earthlyApp) actionSecretsListV2(cliCtx *cli.Context) error {
 		return nil
 	}
 
+	orgName, projectName, isPersonal, err := app.getOrgAndProject(cliCtx.Context, cloudClient)
+	if err != nil {
+		return err
+	}
+
 	for _, secret := range secrets {
 		display := secret.Path
-		if !strings.HasPrefix(display, "/user/") {
-			prefix := fmt.Sprintf("/%s/%s/", app.orgName, app.projectName)
+		if isPersonal && projectName == "" {
+			display = strings.TrimPrefix(display, "/user/")
+		} else {
+			prefix := fmt.Sprintf("/%s/%s/", orgName, projectName)
 			display = strings.TrimPrefix(display, prefix)
 		}
 		fmt.Println(display)
```

Comment:
this form of "initialize a variable, then use a series of logic branches to reassign it" suggests that it belongs in a helper function:

```go
func secretDisplay(personal bool, org, proj string, secret *Secret) string {
    if personal && proj == "" {
        return strings.TrimPrefix(secret.Path, "/user/")
    }
    return strings.TrimPrefix(display, fmt.Sprintf("/%s/%s/", org, proj))
}

// then in this code
fmt.Println(secretDisplay(isPersonal, orgName, projName, secret))
```